beacon-chain/rpc/validator/status.go

```diff
@@ -202,10 +203,14 @@ func (vs *Server) validatorStatus(
 			} else {
 				// Check if there was a deposit deposit.
 				deposit, eth1BlockNumBigInt := vs.DepositFetcher.DepositByPubkey(ctx, pubKey)
-				if eth1BlockNumBigInt != nil && deposit.Data.Amount >= params.BeaconConfig().MaxEffectiveBalance {
-					resp.Status = ethpb.ValidatorStatus_DEPOSITED
-				} else if eth1BlockNumBigInt != nil && deposit.Data.Amount > 0 {
-					resp.Status = ethpb.ValidatorStatus_PARTIALLY_DEPOSITED
+				if eth1BlockNumBigInt != nil {
```

Comment:
Want to add a test for this logic?

Reply:
😅 Okay. I'll have to investigate how to actually get these to trigger.

Reply:
Can always refactor this out into a helper function for testability :)